signac/__main__.py

```diff
@@ -724,16 +722,14 @@ def main_config_show(args):
         cfg = config.read_config_file(config.USER_CONFIG_FN)
     else:
         cfg = config.load_config()
-    if cfg is None:
-        if args.local and args.globalcfg:
-            mode = " local or global "
-        elif args.local:
-            mode = " local "
+    if not cfg:
+        if args.local:
+            mode = "local"
         elif args.globalcfg:
-            mode = " global "
+            mode = "global"
         else:
-            mode = ""
-        _print_err(f"Did not find a{mode}configuration file.")
+            mode = "local or global"
+        _print_err(f"Did not find a {mode} configuration file.")
         return
```

Comment:
Should this exit with some nonzero return code? I notice that some of our code paths calling `_print_err` are informational, while others indicate actual errors and are followed by `sys.exit(1)`. Some code paths raise `RuntimeWarning`, which also exits with code 1. https://github.com/glotzerlab/signac/blob/c8cc307566c78f9f960abc43fb793b6b368ec18d/signac/__main__.py#L1715-L1719

Perhaps we need to assess this in a separate PR, and make sure all the code in `__main__.py` is consistent about using nonzero exit codes for errors. For now I'll mark a couple cases that are obvious.

```suggestion
        sys.exit(1)
```